app/actors/iiif_print/actors/file_set_actor_decorator.rb

```diff
@@ -22,7 +22,7 @@ def create_content(file, relation = :original_file, from_url: false)
           end
         else
           # we don't have the parent yet... save the paths for later use
-          @pdf_paths = service.pdf_paths(files: [file.id.to_s])
+          @pdf_paths = service.pdf_paths(files: [file.try(:id)&.to_s])
```

Comment:
This will result in `[nil]` which `[nil].nil? == false`; so the below guard condition won't work.

To resolve, we want to do the following:
```suggestion
          @pdf_paths = service.pdf_paths(files: [file.try(:id)&.to_s].compact)
```

Which results in the following: `[nil].compact == []`